### DISCUSSION TITLE
Todo comment highlights in pull requests

### DISCUSSION THREAD

**Discussion by mariusbippig** [community/community#75598] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi, I did not find something about another feature request, so I will just post what I have in mind:

I'm using Jetbrains IntellIJ and in my IDE TODO and FIXME comments are highlighted by default with another colour. I would wish to see those comments highlighted in pull requests on Github as well. Currently, they are just grey, like all other comments.
What I don't know is, if this is restricted to just a few languages, but as far as it is a built-in feature of Jetbrains products, I assume it is not.

<img width="315" alt="Screenshot 2023-11-15 at 09 38 51" src="https://github.com/community/community/assets/10549051/7502f708-774a-45d7-aec5-70b88dc251bb">

Thank you!

**Comment by wasimtikki120** [comment]
Your message is clear and to the point. It effectively communicates your feature request for GitHub to highlight TODO and FIXME comments in pull requests, similar to how they are highlighted in JetBrains IntelliJ. The mention of the default color highlighting in JetBrains IntelliJ and the desire to see the same in GitHub pull requests provides a clear context for your request.

If you plan to submit this as feedback to GitHub, you might want to consider sending it through their official channels, such as their support platform or feedback forums. Here's a slight modification for clarity:

---

**Subject: Feature Request - Highlight TODO and FIXME Comments in Pull Requests**

Hi,

I hope this message finds you well. I couldn't find information about this in previous feature requests, so I'm submitting a new one.

I'm a JetBrains IntelliJ user, and I've noticed that TODO and FIXME comments are highlighted by default with a different color in the IDE. I would like to propose extending this feature to GitHub pull requests as well. Currently, these comments appear grey, similar to all other comments.

I'm unsure if this is restricted to specific languages, but given that it's a built-in feature of JetBrains products, I assume it is not.

[Attach Screenshot if applicable]

This enhancement would significantly improve the pull request reviewing experience. If you need more details or if there are similar requests, please let me know.

Thank you for your attention and the excellent work on GitHub!

Best regards,
[Your Name]
